### ISSUE TITLE
Internal Bleeding mechanics

### ISSUE THREAD

**Issue by thomas15v** [thomas15v/RealMedicine#1]
- [x] Made BloodTransfusion and Blood donation operations
- [x] Made BloodLoss replenish slower
- [x] Add Bags and Blood bags
- [ ] Change glitterworld medicine to only be used for internal healing
- [ ] Add operating tools

**Comment by Avexsis** [comment]
I like these ideas alot, except for the third. Internal bleedings do not depent on external and they all can cause death and require surgery.

**Comment by thomas15v** [comment]
Thank you for your input. Il just make internal bleedings go a bit slower (25%). I still would prefer that people can play tribal with my mod, and don't have all their pawns die during their first raid.

Question about internal bleedings. Is there really no way to slow/fix them with medicine?

**Comment by Avexsis** [comment]
No, absolutely not. If internal bleeding is suspected it is treated with surgery in the first place. They are more serious than external.
